index.html

```diff
@@ -19,19 +19,27 @@
                     <span class="nav-bar"></span>
                     <span class="nav-bar"></span>
                 </section>
-                
+                <ul class = "menu-names menu-list color">
+                    <li><a href="#work-section" class="color">Portfolio</a></li>
+                    <li><a href="#work-experience" class="color">About</a></li>
+                    <li><a href="#contact-form" class="color">Contact</a></li>
+                    <li><img src="images/Icon - Mail.png"></li>
                 </nav>
-        </header>
+            </header>
         <main>
+        
         <article class="headline">
             <section class="headline-content">
-                <h1 id="intro-text">Hey There. Iam Alex</h1>
+                <div class="flex-headline">
+                <h1 id="intro-text">Hey There.</h1>
+                <h1 id="intro-text2">Iam Clinton.</h1>
                 <h2 id="description-text">I'm a Software Developer</h2>
                 <p id="welcome-text">I can help you build a product , feature or website 
                     Look through some of my work and experience! 
                     If you like what you see and have a project you need coded, 
                     don’t hesitate to contact me.
-                </p>  
+                </p> 
+            </div>
                 <section class="my-socials">
                     <a href="https://github.com/clin2on3mun"><img src="images/github.png" alt="github"></a>
                     <a href="https://www.linkedin.com/in/munana-clinton-b6122720b/"><img src="images/linkedIn.png" alt="linkedIn"></a>
```

Comment:
- [ ] You did a great job implementing the desktop version of this section, 👍  But I think there is still room for improvement as one of the aims of this project is to replicate the Figma design as much as you can. 🛩️ 
- -  On the screen sizes between `768px` and `900px`, the background color of your main section changes to white, which makes the content of the page unreadable, Please fix this. 😄 
- Screenshot:
![Screenshot 2023-04-01 094020](https://user-images.githubusercontent.com/107632780/229276430-b73d7110-a53e-4d84-9e58-7bf899d8cda5.png)

- - Also on screen sizes between `900px` and `1230px` your background image in this section contrasts with the text on your website, This makes it difficult to read the text. Please consider fixing this. -: U can try reducing the background image size at this screen size 👍 
![Screenshot 2023-04-01 094403](https://user-images.githubusercontent.com/107632780/229276610-a558d438-b882-4093-86fb-d307ada5383d.png)